spec/controllers/prompt_meme_controller_spec.rb

```diff
@@ -2,6 +2,42 @@
 
 describe Challenge::PromptMemeController do
   include LoginMacros
+  include RedirectExpectationHelper
+
+  describe "new" do
+    context "when the collection already has a challenge" do
+      before do
+        @collection = FactoryGirl.create(:collection, challenge: PromptMeme.new)
+        fake_login_known_user(@collection.owners.first.user)
+        post :new, params: { collection_id: @collection.name }
+      end
+
+      it "redirects to edit meme collection path with notice" do
+        it_redirects_to_with_notice(edit_collection_prompt_meme_path(@collection), "There is already a challenge set up for this collection.")
+      end
+    end
+  end
+
+  describe "update" do
+    context "when it fails to update parameters" do
+      before do
+        @collection = FactoryGirl.create(:collection, challenge: PromptMeme.new)
```

Comment:
Just `create(:collection, challenge: PromptMeme.new)` should work here and below. We prefer to [leave out `FactoryGirl` in the specs](https://github.com/otwcode/otwarchive/wiki/Automated-Testing#Our_RSpec_Standards).